src/main/java/com/example/gradecalculator/web/controller/FormController.java

```diff
@@ -1,34 +1,37 @@
 package com.example.gradecalculator.web.controller;
-
-import com.example.gradecalculator.entities.UserType;
-import com.example.gradecalculator.repository.UserTypeRepository;
-import org.apache.catalina.User;
+import com.example.gradecalculator.entities.User;
+import com.example.gradecalculator.repository.UserRepository;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.autoconfigure.security.SecurityProperties;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.ResponseBody;
 
 @Controller
+@SpringBootApplication
 public class FormController {
```

Comment:
please rename this controller to a more appropriate name, e.g. UserController

Reply:
yup, done

Reply:
there are no changes in this PR for this